remove-duplicates-from-sorted-array/index.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * @param {number[]} nums
+ * @return {number}
+ */
+var removeDuplicates = function(nums) {
+    let l = 0, r = 0
+    for (let i = 0; i < nums.length; i++) {
+        if (nums[i] === nums[i+1] ) {
+            r = r + 1;
+        }
+        else {
+            l = l + 1;
+            nums[l] = nums[i+1]
+        }
+    }
+    return l
+}
```

Comment:
Also there is exists case when u can use Guard Clause approach to improve your function.
See more about Guard Clause here - https://betterprogramming.pub/refactoring-guard-clauses-2ceeaa1a9da